tests/python/gaia-ui-tests/gaiatest/apps/system/regions/utility_tray.py

```diff
@@ -41,8 +41,8 @@ def tap_settings_button(self):
         self.marionette.find_element(*self._quicksettings_app_locator).tap()
 
     def a11y_wheel_utility_tray_grippy(self):
-        self.accessibility.wheel(self.marionette.find_element(
-            *self._grippy_locator), 'up')
```

Comment:
Yes, I removed the wrong line(I managed to mix this up with the line that I didn't remove)